### PR TITLE
Add email address

### PR DESCRIPTION
**Pull request by lyda**
Add my email address to my credit line.

Maintainer: @PolynomialDivision
Compile tested: none
Run tested: none

Description: Changed a comment line to add my email address.

### CODE REVIEW

**Comment by BKPepe** [comment]
Hello @lyda,

Thank you for submitting a pull request. I'd love to merge this, but I can not as there are three things that needs to be sorted out.

1. Add [Signed-off-by](https://openwrt.org/submitting-patches) in the commit description as I can not do this on your behalf.
2. Change the commit subject/title to smth like: ``prometheus-node-export-lua``: add my email address to copyright. In the first part, I mean the prefix with the package name is required. The rest is up to you.
3. It is a question if we should bump **PKG_RELEASE** in this case or not.
Basically, what it does that existing installations would be requested to update to this new version just for copyright change. :-( So, it will be applied only for new installations w/o bumping it.

**Comment by PolynomialDivision** [comment]
I'm not the maintainer. @champtar

**Comment by champtar** [comment]
Please fix at least point 1 and 2 mentioned by @BKPepe

**Comment by BKPepe** [comment]
Closing for now, if you do such changes, feel free to re-open.
